lib/matplotlib/_mathtext.py

```diff
@@ -2621,3 +2626,31 @@ def auto_delim(self, s, loc, toks):
             # if "mid" in toks ... can be removed when requiring pyparsing 3.
             toks["mid"].asList() if "mid" in toks else [],
             toks["right"])
+
+    def boldsymbol(self, s, loc, toks):
+        self.push_state()
+        state = self.get_state()
+        small_greek = [unicodedata.name(chr(i)).split()[-1].lower() for i in
+                           range(ord('\N{GREEK SMALL LETTER ALPHA}'),
+                                 ord('\N{GREEK SMALL LETTER OMEGA}') + 1)]
```

Comment:
Maybe one should move this to be a member of the parser class so that it is not recreated every time?

(And considering that it is only used for testing membership, make it a `set`?)